lib/utils/grpc.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"context"
+
+	"github.com/gravitational/trace/trail"
+	"google.golang.org/grpc"
+)
+
+func ErrorConvertUnaryInterceptor(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
```

Comment:
Do you think it's worth having a simple test for both ErrorConvertUnaryInterceptor and ErrorConvertStreamInterceptor?

Reply:
I don't really think so, it's 3 lines of very straightforward code that will be exercised by most of our other tests